src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequence.tsx

```diff
@@ -50,6 +54,9 @@ type Props = {
   onRemoveSequence?: (index: number | null) => void;
 };
 
+const errorTooltipDescription =
+  'Please check that all your sequences are nucleotide or protein, and that they do not contain any invalid characters';
```

Comment:
Why does this error message, which appears in a single input box, say "all your sequences"?

Reply:
Its per the XD in the Jira, not sure if Andrea was trying to keep it consistent but it makes sense to what you are saying, i will just double check with her and update the text. Should be easy to update it for just one box.